src/test/java/dinkplugin/RarityCalculator.java

```diff
@@ -140,7 +150,8 @@ public Transformed transform() {
                     denom = BigDecimal.valueOf(2000);
                     break;
                 default:
-                    String[] parts = StringUtils.split(cleanRarity, '/');
+                    String fraction = cleanRarity.endsWith("%") ? cleanRarity.substring(0, cleanRarity.length() - 1) + "/100" : cleanRarity;
```

Comment:
won't be necessary in next wiki dump; fixed in https://oldschool.runescape.wiki/?title=TzHaar-Mej_%28monster%29&diff=14552108&oldid=14523815 (unless someone makes the same mistake elsewhere)